.circleci/config.yml

```diff
@@ -1268,36 +1352,35 @@ workflows:
       - dev-upload-docker:
           requires:
             - build-distros-linux
-      - cleanup-gcp-resources
-      - cleanup-azure-resources
-      - cleanup-eks-resources
       # Disable until we can use UBI images.
-      # - acceptance-openshift:
-      #    requires:
-      #      - cleanup-azure-resources
+      # - acceptance-openshift
       - acceptance-gke-1-23:
           requires:
-            - cleanup-gcp-resources
             - dev-upload-docker
       - acceptance-gke-cni-1-23:
           requires:
             - acceptance-gke-1-23
       - acceptance-eks-1-21:
           requires:
-            - cleanup-eks-resources
             - dev-upload-docker
       - acceptance-eks-cni-1-21:
           requires:
             - acceptance-eks-1-21
       - acceptance-aks-1-22:
           requires:
-            - cleanup-azure-resources
             - dev-upload-docker
       - acceptance-aks-cni-1-22:
           requires:
             - acceptance-aks-1-22
+      - acceptance-tproxy:
```

Comment:
how come we're adding acceptance-tproxy here? I think the cloud tests run with tproxy enabled right? Plus PRs run this step, so maybe we don't need it in the nightlies?

Reply:
Sure, if we feel they're not needed I'm totally okay with that. Was just going by what we decided on for the testing strategy in the swim lane diagram

Reply:
Actually since its running overnight, it might be nice to have a kind test case at night too, might be a nice comparison point against cloud tests also. (if only clouds fail but kind passes on the same commit)